prov/sockets/src/sock_cntr.c

```diff
@@ -210,8 +210,9 @@ void sock_cntr_inc(struct sock_cntr *cntr)
 {
 	pthread_mutex_lock(&cntr->mut);
 	atomic_inc(&cntr->value);
-	if (atomic_get(&cntr->value) >= atomic_get(&cntr->threshold))
-		pthread_cond_signal(&cntr->cond);
+	/* if(atomic_get(&cntr->waiting)) { */
```

Comment:
I believe I misread the context around this line. I was worried about a race between reading cntr->waiting and signalling the condition variable, but now I see we already have the mutex here.